Formula/gst-plugins-rs.rb

```diff
@@ -20,18 +19,19 @@ class GstPluginsRs < Formula
   depends_on "cargo-c" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
+  depends_on "python@3.11" => :build # needs python with tomllib
   depends_on "rust" => :build
   depends_on "dav1d"
+  depends_on "gst-plugins-bad" # for webrtc
   depends_on "gst-plugins-base"
   depends_on "gstreamer"
   depends_on "gtk4"
   depends_on "pango" # for closedcaption
 
-  # commit ref, https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/commit/ea98a0b5964cd196abbb48c621969a8ef33eb157
-  # remove in next release
-  patch :DATA
-
   def install
+    # Fixes an issue where the wrong Python is picked up in the Linux build
+    ENV.prepend_path "PATH", Formula["python@3.11"].opt_bin
```

Comment:
We no longer provide `python3` in `bin` for non-aliased Pythons.
```suggestion
    ENV.prepend_path "PATH", Formula["python@3.11"].opt_libexec/"bin"
```